pkg/controller/networkpolicy/mutate.go

```diff
@@ -124,6 +125,13 @@ func (m *NetworkPolicyMutator) mutateAntreaPolicy(op admv1.Operation, ingress, e
 			allPaths = append(allPaths, fmt.Sprintf("/spec/tier"))
 			allValues = append(allValues, defaultTierName)
 		}
+		// For ANP, add policy Namespace to ServiceReference that doesn't set Namespace
+		// in ToServices.
+		if npNamespace != "" {
```

Comment:
In ANP, when only podSelector is set, it selects the Pods in the namespace automatically. I'm wondering if we should just set the namespace when converting it to controlplane NetworkPolicy, without the need to mutate in webhook.

Reply:
didn't we add  "namespace" as required field for ANP yaml? so this is not needed

Reply:
I thought only ACNP requires the namespace field. For ANP, if the user didn't set the namespace, we will use the namespace of the policy as the namespace of the Service.
I removed the setting namespace process from mutating webhook and set namespace when converting ANP to controlplane NetworkPolicy.